tests/Unit/Metadata/Service/ClassMetadataSearcherTest.php

```diff
@@ -47,11 +49,19 @@ class ClassMetadataSearcherTest extends TestCase
     /** @var ElasticSearch|MockObject */
     private $elasticSearch;
 
+    /** @var Ontology|MockObject */
+    private $model;
+
+
+    private $classMock;
```

Comment:
Please add annotations.

```suggestion
    /** @var core_kernel_classes_Class|MockObject **/
    private $classMock;
```